server/appbackend/api.py

```diff
@@ -200,27 +182,38 @@ def post(self, request, *args, **kwargs):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
-class LoginByPhoneApi(generics.CreateAPIView):
-    schema = schema = ManualSchema(fields=[
-        coreapi.Field(
-            "phone",
-            required=True,
-            location="form",
-            schema=coreschema.String()
-        ),
-        coreapi.Field(
-            "group",
-            required=True,
-            location="form",
-            schema=coreschema.Integer()
+class CreateLessonApi(generics.CreateAPIView):
+
+    queryset = Lesson.objects.all()
+    serializer_class = []
+
+    @swagger_auto_schema(responses={200: LessonSerializer})
+    def post(self, request, *args, **kwargs):
+        class_group = ClassGroup.objects.get(pk=kwargs["pk"])
+        lesson = Lesson.objects.create(
+            class_group=class_group
+        )
+        return Response(
+            data=LessonSerializer(lesson).data,
+            status=status.HTTP_201_CREATED
         )
-    ], encoding='application/json')
+
+
+class LoginByPhoneApi(generics.CreateAPIView):
+
+    class LoginByPhoneRequest(serializers.Serializer):
+        phone = serializers.CharField()
+        group = serializers.IntegerField()
+
+        class Meta:
+            fields = '__all__'
 
     permission_classes = []
     serializer_class = PersonSerializer
 
     queryset = Person.objects.all()
 
+    @swagger_auto_schema(request_body=LoginByPhoneRequest)
     def post(self, request, *args, **kwargs):
         phone = request.data.get("phone", "")
         group = request.data.get("group", "")
```

Comment:
Shouldn't it return 204 like the LoginApi class?

Reply:
I've defined as a default behavior that collaborators will need email addresses to log in. This may be false, but since we cannot talk to the PO right now, I think we should keep it like this for now.